### ISSUE TITLE
Cristobal: UX/UI Design, 2 weeks

### ISSUE THREAD

**Issue by cristobal-lopez** [lab-brussels-1/home#268]
[My prototype](https://github.com/MMikhailova/ux-ui-design-module)

### [1. Design Thinking](./1-design-thinking)

- [x] 🥚 **User Empathy**: You understand what user empathy is, and can use it
      to understand a classmate's needs and help them plan a home page.
- [x] 🥚 **Design Cycle**: You will be able to describe the design cycle and
      what steps you will take in each state (Empathize, define, ideate,
      prototype, testing and implement).
- [x] 🥚 **Double Diamond**: You can separate the design and deliver phases of a
      project. Design and delivery are a never-ending cycle, but it helps to be
      clear when you are focusing one or the other.
- [x] 🥚 **Stakeholder Interviews**: You can interview another student to learn
      their motivations, needs, and blocks
- [x] 🥚 **Rapid Prototyping**: You can use rapid prototyping techniques (such
      as paper prototyping) to test initial design hypotheses and receive user
      feedback.
- [ ] 🥚 **User Journeys**: You can write user journeys for websites that you
      use.

**Comment by cristobal-lopez** [comment]
## Week 1

## I Need Help With:
- In the previous module, we learned about the different MD files that the planning contains. Is the sketching added in any of those MD files? or is UX/UI not included in the development strategy?


## What went well?
- [x] Discussing as a team what the main issue of the customer in the food truck was, and proposing multiple solutions. 
- [x] Drawing the steps we followed as a team to reach out to a solution.
- [x] Adding screens to the app to be developed to solve the problem.

## What went less well?
- I am not well at drawing in the conventional way, had a hard time sketching. 
[See my prototype](https://github.com/cristobal-lopez/UI-UX-design)

## Lessons Learned
- It is absolutely important to draw/sketch ideas in paper or digitally to have a wide angle of what is going to be built and this way, we do not need to code prior presenting our prototype to the client. 

## Sunday Prep Work
- I will get ready by reading the pre-class material for next week.

**Comment by danielhalasz** [comment]
> * In the previous module, we learned about the different MD files that the planning contains. Is the sketching added in any of those MD files? or is UX/UI not included in the development strategy?

good observation. the process is still the same, just adding new layers on top of it. a bit like shown in the spider-web diagram: 
![image](https://user-images.githubusercontent.com/78535490/165929985-d1e9a3b5-55f8-4167-b2f2-4458c6e636fd.png)
now you can use UX and UI principles during the backlog and design phases of a project, to have a more elaborate and well researched idea before coding starts.

**Comment by cristobal-lopez** [comment]
## Week 2

## I Need Help With:
- If we sketch our prototype in Figma, do we still need to make the design in Excalidraw?

## What went well?
- [ ] Made a questionnaire to interview a classmate acting like a client of a company who needs a website. 
- [x] Interviewed my classmate @Haneefa-Shaik and made an analysis of the information provided.
- [ ] Created a prototype of her website according to her needs.
- [ ] Had a meeting to discuss and present the prototype.
- [ ] Made changes to the prototype and upload it in my repo. 
[Prototype](https://github.com/cristobal-lopez/UX-UI-design-week-2/blob/master/planning/design.md)

## What went less well?
- Figma is a bit challenging now, but after some practice, I am becoming more knowledgeable. 

## Lessons Learned
- It is a good idea to discuss with the client about every details of what they expect to have a good product coded.

## Sunday Prep Work
- I will get ready by reading the pre-class material for next week. Really eager to start with JS.

**Comment by danielhalasz** [comment]
> * If we sketch our prototype in Figma, do we still need to make the design in Excalidraw?

no, Excalidraw is a simple tool for wireframing, while Figma is much more advanced in every aspect



> * Figma is a bit challenging now, but after some practice, I am becoming more knowledgeable.

can you please add a link to your Figma document? I could not find it

**Comment by cristobal-lopez** [comment]
Hi Daniel,

Here are the links to my Figma prototype and to my interview:

-Figma:

[Figma prototype](https://www.figma.com/file/fK7T0Mcik3KIG7NCL7ZCid/Construction-Company-Website-for-Haneefa-Saik?node-id=1392%3A457)

-Interview:

[UX-UI design.docx](https://github.com/lab-brussels-1/home/files/8639577/UX-UI.design.docx)
